lametro/static/css/city_custom.css

```diff
@@ -423,3 +423,20 @@ hr .events-line {
     background-position: center center;
     height: 200px;
 }
+/*
+body {
+    /* text-size-adjust: none; */
+    /* -moz-text-size-adjust: none; */
+    /* font-size: 30px; */
+} */
+
+
+/* @-moz-document url-prefix() {
+    @media only screen and (max-device-width : 415px) and (orientation: portrait) {
+        .blurb {
+            text-size-adjust: none;
+            -moz-text-size-adjust: none;
+            font-size: 30px;
+        }
+    }
+} */
```

Comment:
You can remove this dead code!

```suggestion
```

Reply:
Bah, thanks for catching that! That's what happens when I don't double check myself :expressionless: